src/main/groovy/org/pih/warehouse/importer/ProductSupplierPreferenceExcelImporter.groovy

```diff
@@ -11,9 +11,24 @@ package org.pih.warehouse.importer
 
 import grails.util.Holders
 import org.grails.plugins.excelimport.AbstractExcelImporter
+import org.grails.plugins.excelimport.DefaultImportCellCollector
+import org.grails.plugins.excelimport.ExcelImportService
 import org.grails.plugins.excelimport.ExpectedPropertyType
 
-class ProductSupplierPreferenceImporter extends AbstractExcelImporter {
+class ProductSupplierPreferenceExcelImporter extends AbstractExcelImporter implements DataImporter {
+
+    static cellReporter = new DefaultImportCellCollector()
+
+    ExcelImportService excelImportService
+    @Delegate
+    ProductSupplierPreferenceImportDataService productSupplierPreferenceImportDataService
```

Comment:
same as above (just check all the files, so I won't repeat the same comment throughout all the files)